src/flb_engine.c

```diff
@@ -697,6 +697,13 @@ int flb_engine_start(struct flb_config *config)
             flb_sched_timer_cleanup(config->sched);
             flb_upstream_conn_pending_destroy_list(&config->upstreams);
         }
+
+        /* this log info will make sure the process for clean the expired message
+        * will at least be triggered every 1.5 sec
+        */
+        #ifdef FLB_HAVE_AWS_ERROR_REPORTER
+        flb_info("[engine] error reporting ttl fresh");
+        #endif
```

Comment:
Can you explain why this is needed in detail?

Reply:
The message also might appear strange to a user, what does "error reporting ttl fresh" mean?

Reply:
So currently we don't create a new event for clean up expired message in message queue. And the clean up place will be triggered every time when this's a log event. Adding this log is just to make sure for some plugin which don't have many log could get message clean in time. 
Since output a log might causing customer confused. I refactor this part code to directly call clean up message function so customer should see no log when we do the expired message clean up. That's should be better for user experience.